src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -406,16 +406,15 @@ public Response get(@Context HttpHeaders httpHeaders, String json) throws Except
 
 
 		final Span span = createSpan("datapoints_query", httpHeaders);
-		try (Scope scope = tracer.scopeManager().activate(span, false)) {
+		try (Scope scope = tracer.scopeManager().activate(span)) {
 			File respFile = File.createTempFile("kairos", ".json", new File(datastore.getCacheDir()));
 			BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(respFile), "UTF-8"));
 			JsonResponse jsonResponse = new JsonResponse(writer);
 			jsonResponse.begin();
 			List<QueryMetric> queries = queryParser.parseQueryMetric(json);
 
-
 			return limiter.callWithTimeout(() -> {
-				try (Scope internalScope = tracer.scopeManager().activate(span, false)) {
+				try (Scope internalScope = tracer.scopeManager().activate(span)) {
 
 					for (QueryMetric query : queries) {
 						Map<String, Collection<String>> tags = query.getTags().asMap();
```

Comment:
Does the call to `getTags()` result in accessing the data store?  Or do we get an empty map in case query is marked for rejection?

Reply:
this refers to the tags that were passed in a query by the user... so it doesn't require any access to data store